si/orange3.jaml

```diff
@@ -9904,130 +9904,130 @@ widgets/model/owsavemodel.py:
     __main__: false
 widgets/model/owsgd.py:
     class `OWSGD`:
-        Stochastic Gradient Descent: null
-        'Minimize an objective function using a stochastic ': null
-        approximation of gradient descent.: null
-        icons/SGD.svg: null
-        Orange.widgets.regression.owsgdregression.OWSGDRegression: null
-        sgd: null
+        Stochastic Gradient Descent: Stohastični gradientni spust
+        'Minimize an objective function using a stochastic ': 'Minimiziranje ciljne funkcije z uporabo stohastičnega '
+        approximation of gradient descent.: približka gradientnega spusta.
+        icons/SGD.svg: false
+        Orange.widgets.regression.owsgdregression.OWSGDRegression: false
+        stochastic gradient descent, sgd: stochastic gradient descent, sgd
         class `Outputs`:
             Coefficients: Koeficienti
-        Squared Loss: null
-        squared_error: null
-        Huber: null
-        huber: null
-        ε insensitive: null
-        epsilon_insensitive: null
-        Squared ε insensitive: null
-        squared_epsilon_insensitive: null
-        Hinge: null
-        hinge: null
-        Logistic regression: null
-        log: null
-        Modified Huber: null
-        modified_huber: null
-        Squared Hinge: null
-        squared_hinge: null
-        Perceptron: null
-        perceptron: null
-        None: null
-        none: null
-        Lasso (L1): null
-        l1: null
-        Ridge (L2): null
-        l2: null
-        Elastic Net: null
-        elasticnet: null
-        Constant: null
-        constant: null
-        Optimal: null
-        optimal: null
-        Inverse scaling: null
-        invscaling: null
-        SGD: null
+        Squared Loss: Kvadratna izguba
+        squared_error: false
+        Huber: Huberjeva
+        huber: false
+        ε insensitive: ε neobčutljiva
+        epsilon_insensitive: false
+        Squared ε insensitive: Kvadratna ε neobčutljiva
+        squared_epsilon_insensitive: false
+        Hinge: Izguba največje tolerance
```

Comment:
Mogoče kar: Odmik (hinge). V oklepaju lahko (tu in tudi pri ostalih) puščamo angleške izraze, da bo očitneje, kaj mislimo s tem prevodom.